wwwapp/migrations/0059_migrate_away_from_python_allaccess.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-03-07 06:29
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.conf import settings
+
+# After the migration is applied in production just comment out code depending on the allaccess library
+def forwards_func(apps, schema_editor):
+    AccountAccess = apps.get_model('allaccess', 'AccountAccess')
+    Provider = apps.get_model('allaccess', 'Provider')
+    UserSocialAuth = apps.get_model('social_django', 'UserSocialAuth')
+
+    for access in AccountAccess.objects.all():
+        if access.provider.name == 'facebook':
+            UserSocialAuth(provider='facebook', uid=access.identifier, user=access.user, extra_data=access.access_token).save()
+            access.delete()
+        elif access.provider.name == 'google':
+            UserSocialAuth(provider='google-oauth2', uid=access.identifier, user=access.user, extra_data=access.access_token).save()
+            access.delete()
+        else:
+            raise RuntimeError(f"Got unknown provider: {access.provider.name}! Aborting migration!")
+    Provider.objects.all().delete()
+
+def reverse_func(apps, schema_editor):
+    AccountAccess = apps.get_model('allaccess', 'AccountAccess')
+    Provider = apps.get_model('allaccess', 'Provider')
+    UserSocialAuth = apps.get_model('social_django', 'UserSocialAuth')
+
+    if not settings.SOCIAL_AUTH_GOOGLE_OAUTH2_KEY or not settings.SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET:
```

Comment:
you should be setting these to None if they don't exist in `settings.py` if you want to check them like this here, this fails with `AttributeError: 'Settings' object has no attribute 'SOCIAL_AUTH_GOOGLE_OAUTH2_KEY'`